.github/workflows/ci.yml

```diff
@@ -120,17 +123,68 @@ jobs:
         run: |
           ddev composer install
           ddev pull platform -y
-          ddev drush deploy -y
+
+      - name: Run phpstan
+        run: ddev php ./vendor/bin/grumphp run --tasks=phpstan --no-interaction
+
+      - name: Rebuild cache
+        run: |
+          ddev drush cache:rebuild -y
+          ddev drush config:status
```

Comment:
Was this for debugging? Is it useful to keep these outputs in the logs?

Reply:
This is to rebuild the cache and config cache. It seems cache:rebuild is not rebuilding the config cache. After running config:status, it recaches all configs. We are pulling only the database from Platform.sh, and Redis is empty on DDEV start. So, I am rebuilding all the caches after the pull.